### PR TITLE
Add Fibonacci player to roulette simulator.

### PR DESCRIPTION
**Pull request by FarzanRashid**
This PR adds Fibonacci player to the roulette simulator. This PR also includes unit tests for PlayerFibonacci class.

### CODE REVIEW

File: src/players/player_fibonacci.py
Comment (FarsanRashid) [inline src/players/player_fibonacci.py:1]: file name need to be renamed `fibonacci.py`

File: tests/test_player_fibonacci.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+from players.player_fibonacci import PlayerFibonacci
+
+
+class TestPlayerFibonacci(TestCase):
+    def setUp(self):
+        self.table = Table()
+        self.player_fibonacci = PlayerFibonacci(self.table)
+        self.bet = Bet(1, self.player_fibonacci.outcome)
+
+    def test_win_resets_recent_value(self):
```
Comment (FarsanRashid) [inline tests/test_player_fibonacci.py:13]: test name need to be renamed. e.g `test_recent_value_is_reset_after_win`

File: tests/test_player_fibonacci.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        self.player_fibonacci.recent = 10
+        expected_recent_value_after_win = 1
+
+        self.player_fibonacci.win(self.bet)
+
+        self.assertEqual(self.player_fibonacci.recent, expected_recent_value_after_win)
+
+    def test_win_resets_previous_value(self):
```
Comment (FarsanRashid) [inline tests/test_player_fibonacci.py:21]: same as the other comment related to renaming

**Review by FarsanRashid — CHANGES_REQUESTED** [review]
the test names can be refactored focusing on expressing the behavior, not the technical details. Except that, LGTM.

File: tests/test_player_fibonacci.py (hunk 2 above)
Comment (FarzanRashid) [reply tests/test_player_fibonacci.py:21]: Done.

File: tests/test_player_fibonacci.py (hunk 1 above)
Comment (FarzanRashid) [reply tests/test_player_fibonacci.py:13]: Done.

File: src/players/player_fibonacci.py
Comment (FarzanRashid) [reply src/players/player_fibonacci.py:1]: Done.

**Pull request merged**